Sources/SwiftDocC/SwiftDocC.docc/SwiftDocC/InputDiscovery.md

```diff
@@ -0,0 +1,86 @@
+# Input Discovery
+
+Learn how to discover documentation inputs on the file system.
+
+## Discussion
+
+A ``DocumentationContext/InputsProvider`` discovers documentation catalogs on the file system and creates a ``DocumentationBundle`` from the discovered catalog content.
+
+```swift
+let inputProvider = DocumentationContext.InputsProvider(fileManager: fileSystem)
+
+guard let catalogURL = try inputProvider.findCatalog(startingPoint: startingPoint) else {
+    return
+}
+let bundle = try inputProvider.makeInputs(contentOf: catalogURL, options: BundleDiscoveryOptions())
```

Comment:
Hm... I think for the purposes of this article it's sufficient to see `BundleDiscoveryOptions` being used and showing that additional symbol graph locations are passed in the options. 

If the developer goes to try and discover inputs themselves they would either try to create a new value and discover the first argument for both initializers is some form of Info.plist values, or they would look for an options value that's already passed to their scope and use that.

Reply:
Makes sense. I agree, from the name, it wasn't immediately clear (to me) what `BundleDiscoveryOptions` is. But you're right -- it would hopefully become clearer when the developer goes to discover inputs.